src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -139,8 +146,10 @@ - (void)finishWithError:(NSError *)errorOrNil {
   _self = nil;
```

Comment:
> Maybe set the default to 1 minute, with a header comment documenting that the client programmer may need to change this.

TVL is strongly against us documenting the very possibility of setting a timeout :) His experience is people tend to set timeouts like 30s that work fine in a Sillicon Valley office, but are unrealistically low for developing countries.

I think the best option is indeed to document clearly that calls in-flight retain themselves, and that if it becomes too hard on memory for someone's use case, they can keep a reference to the call object and cancel it when they receive a memory warning.